tools/clang/test/HLSLFileCheck/validation/val-wave-failures-ps.hlsl

```diff
@@ -1,8 +1,8 @@
 // RUN: %dxc -E main -T ps_6_0 %s | FileCheck -input=stderr %s
 
-// CHECK:warning: Gradient operations are not affected by wave-sensitive data or control flow.
-// CHECK:warning: Gradient operations are not affected by wave-sensitive data or control flow.
-// CHECK:warning: Gradient operations are not affected by wave-sensitive data or control flow.
+// CHECK: 37:7: warning: Gradient operations are not affected by wave-sensitive data or control flow.
+// CHECK: 30:16: warning: Gradient operations are not affected by wave-sensitive data or control flow.
+// CHECK: 9:16: warning: Gradient operations are not affected by wave-sensitive data or control flow.
```

Comment:
I wonder. Is the ordering consistently in reverse? If so, iterating in reverse order might give better results

Reply:
The other test seems to have normal order

Reply:
No, I guess. The other test file `WaveSensitivityAndCycles.hlsl` in this PR has warnings printed in ascending order by line number.

Reply:
So I see, bummer 😞